src/components/Menu/Menu.js

```diff
@@ -1,13 +1,36 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import "./menu.css";
 import Button from "../Buttons/Button";
 import { Link } from "react-router-dom";
+import menuIcon from "../../assets/menu-hamburger.svg";
 
 export default function Menu() {
+  const [toggle, setToggle] = useState("close");
```

Comment:
useState("false")

Reply:
-I have implemented the state as a className. So if I change it to boolean value then the code won't be able to change the css for the navbar.
Should I change the css for it as well ??